### PR TITLE
Refactor/recoil to jotai

### PR DESCRIPTION
**Pull request by NdekoCode**
## Refactor/recoil to jotai

Please include a summary of the changes and the related issue.

## Type of Change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Breaking change
-   [ ] Documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings

## Previous screenshots

Please add here videos or images of previous status

## Current screenshots

Please add here videos or images of previous status

### CODE REVIEW

**Comment by Cedric921** [comment]
I'm reviewing this PR

**Comment by evereq** [comment]
@NdekoCode can you VERY carefully please fix merge conflicts?

**Comment by evereq** [comment]
@NdekoCode sorry, can you do one more time fix of merge conflict? 
@Cedric921 did you reviewed PR?

**Comment by NdekoCode** [comment]
Already done sir @evereq, please @Cedric921, try to see the PR again

File: apps/web/package.json [hunk 1]
```diff
@@ -120,6 +120,7 @@
 		"react-popper-tooltip": "^4.4.2",
 		"react-resizable-panels": "^2.0.19",
 		"recoil": "^0.7.7",
```
Comment (evereq) [inline apps/web/package.json:122]: @NdekoCode if you switched from recoil to jotai, why you did not removed "recoild" package? Or NOT all is fixed yet?

**Pull request merged**